themes-default/slim/views/vue-components/anidbReleaseGroupUi.mako

```diff
@@ -0,0 +1,52 @@
+<script type="text/x-template" id="anidb-release-group-ui">
+    <div id="anidb_release_group_ui_wrapper" class="top-10">
+        <link rel="stylesheet" type="text/css" href="css/vue/anidbreleasegroupui.css?${sbPID}" />
```

Comment:
Wouldn't it make more sense to inline the CSS since we want a single vue file later on?

Reply:
good point